app/CocoaPods/CPUserProject.m

```diff
@@ -110,12 +110,12 @@ - (BOOL)writeToURL:(NSURL *)absoluteURL
 
 + (NSSet *)keyPathsForValuesAffectingProgressButtonTitle;
 {
-  return [NSSet setWithObject:@"task"];
+  return [NSSet setWithObject:@"task.progress.fractionCompleted"];
 }
 
 - (NSString *)progressButtonTitle;
 {
-  return self.task == nil ? @"Done" : @"Cancel";
+  return self.task.progress.fractionCompleted == 1.0f ? @"Done" : @"Cancel";
```

Comment:
`fractionCompleted` is a double and not a float this the `1.0f` shouldn't have an `f`